kuma/javascript/src/signin-link.jsx

```diff
@@ -0,0 +1,70 @@
+//@flow
+import * as React from 'react';
+import { useContext } from 'react';
+
+import { getLocale, gettext } from './l10n.js';
+import GAProvider from './ga-provider.jsx';
+
+type Props = {
+    className?: string,
+    text?: string,
+};
+
+export default function SignInLink({
+    className,
+    text = gettext('Sign in'),
+}: Props): React.Node {
+    const ga = useContext(GAProvider.context);
+    const locale = getLocale();
+
+    if (typeof window === 'undefined') {
+        return null;
+    }
```

Comment:
So this is a new thing? How does it relate to subscription management?

Either way, the `Login` component already does this check so `<SigninLink/>` won't be called if `typeof window === 'undefined'` anyway.

Reply:
Good point-- removed this. And since the management page is not SSR, window will always be defined.

Reply:
This relates to the subscription management page, because when you're not logged in, you see this message: 
<img width="393" alt="Screen Shot 2020-04-28 at 9 25 01 PM" src="https://user-images.githubusercontent.com/650/80553153-ca476e00-8996-11ea-924f-9d0908a7faca.png">

And I wanted to re-use what the sign in link in the header was doing, so I moved it into a separate signin-link component.